src/pages/Dashboard/Dashboard.vue

```diff
@@ -124,24 +118,6 @@ export default {
     tabs() {
       return [...serverTabs, ...(this.isCloud ? cloudTabs : [])]
     },
-    usageTile() {
-      if (!this.isCloud) return null
-      if (!this.license) return 'loading'
-      const isSelfServe = this.license.terms.is_self_serve
-      const isUsageBased = this.license.terms.is_usage_based
-
-      // Legacy license, not self-serve (so no upgrade)
-      if (!isSelfServe && !isUsageBased) return null
-
-      // Legacy license, self-serve (so can upgrade)
-      if (isSelfServe && !isUsageBased) return 'UpgradeUsageTile'
```

Comment:
I think it'd still be good to show the upgrade tile here (and on the account page)

Reply:
should be good now :)